lib/devicetrust/native/device_windows.go

```diff
@@ -511,10 +542,131 @@ func solveTPMEnrollChallenge(
 	}, nil
 }
 
+// activateCredentialInElevated child uses `runas` to trigger a child process
+// with elevated privileges. This is necessary because the process must have
+// elevated privileges in order to invoke the TPM 2.0 ActivateCredential
+// command.
+func activateCredentialInElevatedChild(
+	encryptedCredential attest.EncryptedCredential,
+	credActivationPath string,
+	debug bool,
+) ([]byte, error) {
+	exe, err := os.Executable()
+	if err != nil {
+		return nil, trace.Wrap(err, "determining current executable path")
+	}
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil, trace.Wrap(err, "determining current working directory")
+	}
+
+	// Clear up the results of any previous credential activation
+	if err := os.Remove(credActivationPath); err != nil {
+		err := trace.ConvertSystemError(err)
+		if !trace.IsNotFound(err) {
+			return nil, trace.Wrap(err, "clearing previous credential activation results")
+		}
+	}
+
+	// Assemble the parameter list. We encoded any binary data in base64.
+	// These parameters cause `tsh` to invoke HandleTPMActivateCredential.
+	params := []string{
+		"device",
+		"tpm-activate-credential",
+		"--encrypted-credential",
+		base64.StdEncoding.EncodeToString(encryptedCredential.Credential),
+		"--encrypted-credential-secret",
+		base64.StdEncoding.EncodeToString(encryptedCredential.Secret),
+	}
+	if debug {
+		params = append(params, "-d")
```

Comment:
```suggestion
		params = append(params, "--debug")
```

If possible, I prefer the long form as it's more self-documenting.